css/index.css

```diff
@@ -0,0 +1,35 @@
+@import url(https://fonts.googleapis.com/css2?family=Oswald:wght@300&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap);
+
+body{
+    background-color: rgb(165, 243, 243);
+    color: rgb(30, 4, 4);
+    font-family: Roboto, arial;
+    
+}
+#wrapper{
+    width: 80%;
+    margin: auto;
+    text-align: center;
+}
+section{
+    margin : 20px;
+    padding: 20px;
+    
+}
+img{
+    border-radius: 20px;
+}
+h1{
+    font-size: 50px;
+    font-weight: 500px;
+    color: rgb(231, 39, 116);
+    text-align: center;
+}
+h2{
+    font-size: larger;
+    font-weight: 300px;
+    color: rgb(235, 94, 151);
+}
+.fa{
+    width: 50px;
+}
```

Comment:
The styling overall looks great. I would however add the following in order to center the bullet points along with the text inside the unorder list. This can be done with the following code:

ul {
  display: inline-block;
}

li {
  text-align: left;
}